html/shared/navbarSearchItem.html

```diff
@@ -4,7 +4,11 @@
       <i class="fa fa-file-o fa-2x"></i>
     </div>
     <div class="media-body">
-      <div ng-bind-html="match.model.title | uibTypeaheadHighlight:query"></div>
+      <kramjax
+        body="match.model.title | uibTypeaheadHighlight:query"
+        suppress-paragraphs="true"
+        >
+      </kramjax>
```

Comment:
`<kramjax>` should be wrapped in a `<div>`. Otherwise the authors are rendered on the same line:

![screenshot from 2016-04-01 12-26-05](https://cloud.githubusercontent.com/assets/1874116/14204511/0c57ad50-f805-11e5-8260-ea6d18453b7b.png)